app/controllers/new-signup.js

```diff
@@ -59,12 +69,56 @@ export default class NewSignUpController extends Controller {
 
   @action handleInputChange(key, value) {
     this.error = '';
-    set(this.signupDetails, key, value);
+    if (key === 'role') {
+      if (value === 'developer') {
+        this.developer = true;
+        this.designer = false;
+        this.mavens = false;
+        this.productManger = false;
+      } else if (value === 'designer') {
+        this.developer = false;
+        this.designer = true;
+        this.mavens = false;
+        this.productManger = false;
+      } else if (value === 'mavens') {
+        this.developer = false;
+        this.designer = false;
+        this.mavens = true;
+        this.productManger = false;
+      } else if (value === 'productManager') {
+        this.developer = false;
+        this.designer = false;
+        this.mavens = false;
+        this.productManger = true;
+      }
```

Comment:
can we use object here which is initialized to false, and upon selection the value of selected key is made true?

Reply:
Done can you please check my latest commit